Scripts/SL-Histogram.lua

```diff
@@ -39,10 +39,7 @@ NPS_Histogram = function(player, _w, _h)
 			if (PeakNPS and NPSperMeasure and #NPSperMeasure > 1) then
 
 				local TimingData = Steps:GetTimingData()
-
-				-- Don't use Song:MusicLengthSeconds() because it includes time
-				-- at the beginning before beat 0 has occurred
```

Comment:
I think this comment is left on here accidently, because `TotalSeconds` is no longer used.

Reply:
I think you're correct.  Thanks. 👍